pkg/client/didexchange/client.go

```diff
@@ -35,19 +35,19 @@ var ErrConnectionNotFound = errors.New("connection not found")
 type provider interface {
 	Service(id string) (interface{}, error)
 	LegacyKMS() legacykms.KeyManager
-	InboundTransportEndpoint() string
```

Comment:
It would be nice to add a comment to `InboundTransportEndpoint` regarding when to use it (and I assume to prefer ServiceEndpoint?).

Reply:
[Added](https://github.com/hyperledger/aries-framework-go/compare/1845bf271e498ea6bda19e69e21961bf0c0bf0d1..cdfef7229d58c9c2bcc938d50f1e7c2d1c9bb964#diff-9988f5820bf05228e5657f2842fa071eR115).